karoo_gp/test/test_base.py

```diff
@@ -0,0 +1,53 @@
+import hashlib
+
+import pytest
+
+from karoo_gp import Base_GP
+
+def kwargs():
```

Comment:
```suggestion
@pytest.fixture
def kwargs():
```
Here you still need the decorator.